tic-tac-toe.js

```diff
@@ -1,7 +1,91 @@
-function TicTacToe() {
+$(document).ready(function () {
+  var game = new TicTacToe();
+  var cells = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 
-}
+  cells.forEach(function(cell) {
+    $square = $("#cell" + cell);
+    $square.one( "click", (function () {
+      $(this).css("background-color", "#16A765");
+       var id = $(this).attr("id");
+      if (game.turns % 2 === 0) {
+        $(this).css("background-color", "#000");
+      }
+      game.play(id);
+    }));
+  });
+});
 
-TicTacToe.prototype = {
-  
+function TicTacToe() {
+  this.player = "Player 1";
+  this.turns = 1;
+  this.board = [[0, 1, 2], [3, 4, 5], [6, 7, 8]];
+  this.finish = false;
 }
+
+TicTacToe.prototype.turn = function(id) {
+  var block = id[4] - 1;
```

Comment:
You can also use the `splice` method with a negative param which will give you the last element of a string like: `id.splice(-1)`